features.tex

```diff
@@ -22,3 +23,4 @@
 % combinatorics/group theory, code generation, lie algebras, cryptography,
 % category theory, special functions, sets, matrix expressions, series, or
 % vectors.
+\subsection{Other features}
```

Comment:
Series, in particular about Formal Power Series. Also, can help out with the current series expansion infrastructure.(I do not have a lot of idea about ``rs_series`` which should/can be talked about.)